lib/instrumentation/core/http.js

```diff
@@ -413,37 +410,34 @@ function urlToOptions(_url) {
 }
 
 function wrapRequest(agent, request) {
-  // URL as a global was not added until Node v10
-  const URLClass = global.URL ? global.URL : url.URL
+  const URLClass = global.URL
+
   return function wrappedRequest(input, options, cb) {
     let reqArgs = arguments
```

Comment:
I think reqArgs was defined here and then in the if block, redefined. Now that the if block is gone, I think reqArgs can just be defined below on 438

Reply:
Good catch!  I'll update.